tests/simple-node-nixops.nix

```diff
@@ -0,0 +1,17 @@
+import <nixpkgs/nixos/tests/make-test.nix> ({ pkgs, ... }: {
+  name = "boot";
+  nodes = {
+    machine = { config, pkgs, ... }: {
+      imports = [ ../../modules/cardano-node-config.nix <nixops/nix/options.nix> <nixops/nix/resource.nix> ];
```

Comment:
you'll need to call `(import ./../modules/cardano-node-config.nix 0 "")` or refactor the module not to be a sloppy function :)